server/src/controllers/auth.controller.ts

```diff
@@ -23,8 +23,8 @@ export class AuthController {
 
   @Post('login')
   async login(
-    @Body() loginCredential: LoginCredentialDto,
     @Res({ passthrough: true }) res: Response,
+    @Body() loginCredential: LoginCredentialDto,
     @GetLoginDetails() loginDetails: LoginDetails,
```

Comment:
Is that change purely visual, or did you find an issue with the old param order?

Reply:
Just for testing the expected values it's a bit simpler if it is the first one. Order doesn't matter since it's injected anyways.